srv/_modules/cortx_unsupported_features.py

```diff
@@ -0,0 +1,197 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+# How to test:
+# $ salt-call saltutil.clear_cache
+# $ salt-call saltutil.sync_modules
+
```

Comment:
Have you missed to add the actual test command here?

Reply:
Actual test command is tricky. However, I'll include the test process here.

Reply:
Updated the basic commands necessary for testing.

Reply:
Thanks!! 
There are some codacy warnings, please address.